apps/remix-ide/src/app/tabs/locales/en/udapp.json

```diff
@@ -67,16 +67,16 @@
   "udapp.tooltipText3": "Click to open a bridge for converting L1 mainnet ETH to the selected network currency.",
 
   "udapp._comment_instanceContainerUI.tsx": "libs/remix-ui/run-tab/src/lib/components/instanceContainerUI.tsx",
-  "udapp.deployedContracts": "Deployed/Unpinned Contracts",
-  "udapp.deployAndRunClearInstances": "Clear instances list and reset recorder",
+  "udapp.deployedContracts": "Deployed Contracts",
+  "udapp.deployAndRunClearInstances": "Delete all deployed contracts (including pinned) and reset recorder",
```

Comment:
I think the word Delete is not correct - I think it should be:
Clear deployed contract...

Reply:
Delete sounds like it will delete them from a public chain - if connected.  Of course if the RemixVM is connected the word delete would be correct.

Reply:
updated